src/components/v2v3/shared/SplitItem/index.tsx

```diff
@@ -0,0 +1,119 @@
+} from 'constants/styles/colors'
+
+import { Split } from 'models/splits'
+import { twJoin } from 'tailwind-merge'
+import { OldSplit } from 'utils/splits'
+
+import { isJuiceboxProjectSplit } from 'utils/v2v3/distributions'
+import { ETHAddressBeneficiary } from './EthAddressBeneficiary'
+import { JuiceboxProjectBeneficiary } from './JuiceboxProjectBeneficiary'
+import { LockedUntilValue } from './LockedUntilValue'
+import { ReservedTokensValue } from './ReservedTokensValue'
+import { SplitValue } from './SplitValue'
+
+export type SplitProps = {
+  split: Split
+  totalValue: BigNumber | undefined
+  projectOwnerAddress: string | undefined
+  reservedRate?: number
+  valueSuffix?: string | JSX.Element
+  valueFormatProps?: { precision?: number }
+  currency?: BigNumber
+  showFees?: boolean
+  showAmount?: boolean
+}
+
+export function SplitItem({
+  split,
+  showFees,
+  totalValue,
+  projectOwnerAddress,
+  reservedRate,
+  valueSuffix,
+  valueFormatProps,
+  currency,
+  showAmount,
+}: {
+  split: Split & { oldSplit?: OldSplit }
+  totalValue: BigNumber | undefined
+  projectOwnerAddress: string | undefined
+  reservedRate?: number
+  valueSuffix?: string | JSX.Element
+  valueFormatProps?: { precision?: number }
+  currency?: BigNumber
+  showFees?: boolean
+  showAmount?: boolean
+}) {
+  const isJuiceboxProject = isJuiceboxProjectSplit(split)
+
+  const oldSplit = split.oldSplit
+  const splitIsRemoved = oldSplit === true
+  const splitIsNew = oldSplit === false
+
+  const hasDiff = oldSplit !== undefined && !(splitIsRemoved || splitIsNew)
+
+  const className = twJoin(
+    'flex flex-wrap items-baseline justify-between',
+    splitIsRemoved ? DIFF_OLD_BACKGROUND : undefined,
+    splitIsNew ? DIFF_NEW_BACKGROUND : undefined,
+    splitIsRemoved || splitIsNew ? '-ml-5 pr-1' : undefined,
+  )
+
+  return (
+    <div className={className}>
+      <div>
+        <div className="items-baselineleading-6 flex">
+          {splitIsRemoved ? <DiffMinus /> : null}
+          {splitIsNew ? <DiffPlus /> : null}
+          {isJuiceboxProject ? (
+            <JuiceboxProjectBeneficiary
+              projectOwnerAddress={projectOwnerAddress}
+              split={split}
+              oldSplit={hasDiff ? oldSplit : undefined}
+            />
+          ) : (
+            <ETHAddressBeneficiary
+              projectOwnerAddress={projectOwnerAddress}
+              split={split}
+            />
+          )}
+        </div>
+
+        {(split.lockedUntil && split.lockedUntil > 0) || hasDiff ? (
+          <LockedUntilValue
+            lockedUntil={split.lockedUntil}
+            oldLockedUntil={hasDiff ? oldSplit.lockedUntil : undefined}
+            showDiff={hasDiff} // need for LockedText because lockedUntil can be undefined
+          />
+        ) : null}
+      </div>
+      <div className="flex items-center whitespace-nowrap">
+        <SplitValue
+          split={split}
+          oldSplit={hasDiff ? oldSplit : undefined}
```

Comment:
**suggestion**, again, think we can have a `showDiff` prop on this `SplitValue` component, rather than determining whether there is a diff based on `oldSplit.